TFT/src/User/API/menu.c

```diff
@@ -1255,55 +1254,63 @@ void loopBackEnd(void)
 
   // Temperature monitor
   loopCheckHeater();
+
   // Fan speed monitor
   loopFan();
+
   // Speed & flow monitor
   loopSpeed();
 
+  // Buzzer handling
   #ifdef BUZZER_PIN
-    // Buzzer handling
     loopBuzzer();
   #endif
 
+  // Handle a print from (remote) onboard media, if any
```

Comment:
Here the first alphabet is changed to capital.

Reply:
as the other existing comments, so I didn't change them. See my first comment